core/modules/locale/locale.pages.inc

```diff
@@ -124,18 +124,23 @@ function _locale_translate_seek() {
 /**
  * List languages in search result table
  */
-function _locale_translate_language_list($translation, $limit_language) {
+function _locale_translate_language_list($string, $limit_language) {
```

Comment:
Oh, I just realized this is because this change was also made in the upstream patch in D7. Very good; no need to change back. Maintaining the consistency is what had been my concern.